components/x-eventpromo/src/Details.jsx

```diff
@@ -0,0 +1,23 @@
+import { h } from '@financial-times/x-engine';
+import Footer from './Footer';
+import Meta from './Meta';
+
+const Details = ({link, title, ...props}) => {
+
+	return (
+		<div className="event-promo-inarticle__blocks" aria-labelledby="details-header">
+			<h1 id="details-header" style={{display: 'none'}}>Event details and information</h1>
```

Comment:
This isn't the main page headline. Without knowing the context it will sit in it's hard to choose the right level though, but as it's used for labeling rather than a landmark (display none will remove it from the accessibility tree) a `<div>` will probably do, `<div hidden>` would also work.